### ISSUE TITLE
p-multiSelect, p-inputNumber, p-treeTable Accessibility Issues

### ISSUE THREAD

**Issue by geromegrignon** [openng-org/optimus-ui#346]
### Describe the bug

The p-multiSelect, p-inputNumber, p-treeTable components has accessibility violations when running the IBM Equal Accessibility Checker. The issue is: aria specification violations

### Reason for not contributing a PR

- [x] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://stackblitz.com/edit/stackblitz-starters-2eg7tx8m

### Environment

Stackblitz

### Angular version

20.0.4

### PrimeNG version

v20

### Steps to reproduce the behavior

1. Install the [IBM Equal Accessibility Checker](https://chromewebstore.google.com/detail/ibm-equal-access-accessib/lkcagbfjnkomcinoddgooolagloogehp)
2. Open the stackblitz
3. Run the accessibility checker
4. Filter on violations only and note the errors: 
     a. Form control with "combobox" role has no associated label
     b. Form control with "spinbutton" role has no associated label
     c. The ARIA attributes "aria-level, aria-checked" are not valid for the element <tr> with implicit ARIA role "row"

### Expected behavior

There are no accessibility violations for the p-multiSelect, p-inputNumber, p-treeTable components

> Migrated from `primefaces/primeng#18771` — originally by `@srujana10499` on 2025-08-17

**Comment by geromegrignon** [comment]
*[Originally posted by `@github-actions[bot]` on 2026-02-14T00:27:31.000Z]*

This issue has been automatically marked as stale due to a lack of activity. **If this issue is still relevant to you, please leave a comment** so we can keep it open. We apologize for not being able to prioritize it sooner. If you have any new information or questions, please share them in your comment!
